packages/flutter/lib/src/rendering/proxy_box.dart

```diff
@@ -2927,11 +2927,21 @@ class RenderIgnorePointer extends RenderProxyBox {
   RenderIgnorePointer({
     RenderBox child,
     bool ignoring = true,
-    bool ignoringSemantics,
+    bool ignoringSemantics = false,
   }) : _ignoring = ignoring,
+       _wasIgnoring = ignoring,
        _ignoringSemantics = ignoringSemantics,
        super(child) {
     assert(_ignoring != null);
+    assert(_ignoringSemantics != null);
+    if (ignoring && !_ignoringSemantics) {
+      updateMutedSemanticsActions(
+        <SemanticsAction>{
+          SemanticsAction.tap,
```

Comment:
Why only block tap and longPress? I think a RenderIgnorePointer also blocks scrolling and other gestures, no?

Reply:
We will probably want to block more, This is more like a place holder for now